extra-network/shadowsocks-go/autobuild/build

```diff
@@ -0,0 +1,6 @@
+wget http://dl.chenyufei.info/shadowsocks/${PKGVER}/shadowsocks-{local,server}-linux64-${PKGVER}.gz
+gzip -d shadowsocks-local-linux64-${PKGVER}.gz
+gzip -d shadowsocks-server-linux64-${PKGVER}.gz
+install -Dm755 shadowsocks-local-linux64-${PKGVER} ${PKGDIR}/usr/bin/sslocal
```

Comment:
@Arthur2e5 frankly a source build of Go programs is not necessary, but marking TODO for this.